lib/galaxy/auth/providers/ldap_ad.py

```diff
@@ -19,6 +19,38 @@ def _get_subs(d, k, params):
     return str(d[k]).format(**params)
 
 
+def _parse_ldap_options(ldap, options_unparsed):
+    # Tag is defined in the XML but is empty
+    if not options_unparsed:
+        return []
+
+    if "=" not in options_unparsed:
+        log.error("Invalid syntax in <ldap-options>. Syntax should be option1=value1,option2=value2")
+        return []
+
+    ldap_options = []
+
+    # Valid options must start with this prefix. See help(ldap)
+    prefix = "OPT_"
+
+    for key, value in [name.split("=") for name in options_unparsed.split(",")]:
+        try:
+            for n in (key, value):
+                if not n.startswith(prefix):
+                    raise ValueError
+
+            k = getattr(ldap, key)
+            v = getattr(ldap, value)
```

Comment:
Does it make sense to check the prefix? It seems redundant with the `getattr` checks.